app/main.py

```diff
@@ -2,24 +2,79 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: type, owner: type) -> int:
+        if instance is None:
+            return self
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: type, value: int) -> int:
+        if not isinstance(value, int):
+            raise TypeError
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    age = IntegerRange(0, 120)
+    weight = IntegerRange(0, 300)
+    height = IntegerRange(50, 250)
+
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, height: int, weight: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
+
+    def __init__(self, age: int, height: int, weight: int) -> None:
+        self.age = age
+        self.height = height
+        self.weight = weight
```

Comment:
The `ChildrenSlideLimitationValidator` class overrides the `__init__` method but does not call the superclass `SlideLimitationValidator`'s `__init__` method. This can lead to incorrect initialization of the parent class attributes. Consider using `super().__init__(age, height, weight)` to ensure proper initialization.